core/services/vrf/delegate_test.go

```diff
@@ -78,10 +78,10 @@ func buildVrfUni(t *testing.T, db *sqlx.DB, cfg chainlink.GeneralConfig) vrfUniv
 	btORM := bridges.NewORM(db, lggr, cfg.Database())
 	txm := txmmocks.NewMockEvmTxManager(t)
 	ks := keystore.NewInMemory(db, utils.FastScryptParams, lggr, cfg.Database())
+	jrm := job.NewORM(db, prm, btORM, ks, lggr, cfg.Database())
+	t.Cleanup(func() { jrm.Close() })
```

Comment:
Can simplify:
```suggestion
	t.Cleanup(jrm.Close)
```
Or is there an error to check?